src/endpoints/pathogenRepos.js

```diff
@@ -0,0 +1,158 @@
+import { fetch, paginatedFetch } from "../fetch.js";
+import { BadRequest, InternalServerError, NotFound } from '../httpErrors.js';
+import { contentTypesProvided } from "../negotiate.js";
+import { uri } from '../templateLiterals.js';
+import { slurp } from '../utils/iterators.js';
+
+
+const headers = {
+  "Authorization": process.env.GITHUB_TOKEN
```

Comment:
Confirmed for my own sanity that the `GITHUB_TOKEN` we use has no additional permissions, so that this should not expose any private repos.

Reply:
Yep, it's the existing token we use for other unprivileged GitHub requests. Thanks for checking!